docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -35,72 +35,19 @@ Note that the **Script** area works according to the selected language variable.
 
 ## Custom Script FAQ
 
-### How to change JAVA version
-
-If you want to change the JAVA version for your Android project, you can achieve this by changing the `JAVA_HOME` environment variable.
+### How to change Java version
 
 Appcircle currently has `OpenJDK 17` (default), `OpenJDK 8`, `OpenJDK 11` and `OpenJDK 21`.
 
 [Android Build](/workflows/android-specific-workflow-steps/android-build) step uses `OpenJDK 17` as default JDK version.
 
-You can use the below custom script before your build step to change your `JAVA_HOME` environment variable.
-
-```bash
-echo "Default JAVA" $JAVA_HOME
-
-echo "OpenJDK 8" $JAVA_HOME_8_X64
-echo "OpenJDK 11" $JAVA_HOME_11_X64
-echo "OpenJDK 17" $JAVA_HOME_17_X64
-echo "OpenJDK 21" $JAVA_HOME_21_X64
-
-# Change JAVA_HOME to OPENJDK 17
-echo "JAVA_HOME=$JAVA_HOME_17_X64" >> $AC_ENV_FILE_PATH
-```
-
-Create a custom script like above and put it **above** your [Android Build](/workflows/android-specific-workflow-steps/android-build) step.
-
-<Screenshot url="https://cdn.appcircle.io/docs/assets/workflow-android-change-java-workflow.png" />
+To switch JDK versions, you can now use the dedicated **Change Java Version** component, so there is no need to use **Custom Script** for this task. For further details on this component, refer to the documentation:
```

Comment:
```suggestion
To switch JDK versions, you can now use the dedicated [**Select Java Version**](/workflows/common-workflow-steps/change-java-version) component, so there is no need to use **Custom Script** for this task. For further details on this component, refer to the documentation:
```

Reply:
Sorry I forgot to send the last commit I made. That commit includes this change, so I'm closing it.